index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>  <!-- preferrably an h1 tag than a p tag-->
+      <p>The man who saved a billion lives</p>
+      <figure id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt=""
+        />
+        <div id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </div>
+      </figure>
+      
+      <section id="descrition"> <!-- Use of section tag rather than the div tag -->
```

Comment:
Hello, a section tag would be better for this part

Reply:
- [ ] As I suggest above, kindly formulate your comment clearer, giving also the reason why you require changes following the[ ["How to give a code review?"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) course :recycle:

Reply:
Excellent work creating the unordered list. Kindly follow the best semantic tags in HTML. Change the article for a section.